flytekit/remote/remote.py

```diff
@@ -827,8 +828,42 @@ def register_workflow(
         :param serialization_settings: The serialization settings to be used
         :param default_launch_plan: This should be true if a default launch plan should be created for the workflow
         :param options: Additional execution options that can be configured for the default launchplan
+        :param fast: fast register if true
         :return:
         """
+        if fast:
```

Comment:
cc @samhita-alla @cosmicBboy 
Which one do you prefer?
1. remote.register_workflow(my_task, fast=True)
2. remote.fast_register_workflow(my_task)

Reply:
another option might be `remote.register_workflow(my_task, source_root="./")`

Reply:
I do not think like using `fast` here because we already use "fast" with "fast registration". In this case, it's just calling `self.register_script` by defining `source_path` and `module_name` from `entity`. I'll prefer another method all together.